### DISCUSSION TITLE
Table - alternate row color

### DISCUSSION THREAD

**Discussion by andreapx** [ToolJet/ToolJet#6998] [Q&A] [answered]
Hi!
I have a table filled with data from a restAPI, how can I have the color of the row to be alternate (white/light gray, to be more readable)?
Thanks!

**Comment by tejakummarikuntla** [comment, marked as answer]
Hey @andreapx , setting the `Table type` field to `Striped` in the Styles tab of the component will set the table with alternative color styling to the rows.

![Screenshot 2023-08-03 at 12 37 40 PM](https://github.com/ToolJet/ToolJet/assets/34749692/11d2721b-e270-4a6d-a4bd-3a45b9cd7646)

**Comment by andreapx** [comment]
Thanks!
